lemarche/conversations/models.py

```diff
@@ -1,12 +1,17 @@
 from django.conf import settings
 from django.db import models
+from django.db.models import Func, IntegerField
 from django.utils import timezone
 from django_extensions.db.fields import ShortUUIDField
 from shortuuid import uuid
 
 
 class ConversationQuerySet(models.QuerySet):
-    pass
+    def has_answer(self):
```

Comment:
```suggestion
    def with_answers(self):
```
Je te suggère de modifier le nom ou de modifier la nature de la méthode, pcq avec `has_answer` je m'attend à un retour `True` ou `False`

Reply:
yes ca fait sens !
on a eu tendance à utiliser `has_*` dans les queryset, faudrait changer ca dans une autre PR.
le seul hic avec ca, c'est qu'on a aussi des `with_` qui servent à faire des annotation (au lieu de filtrer), faudrait leur trouver un meilleur naming ?

Reply:
je vais merger, j'ai lancé la discussion sur nos canaux, pour trouver un meilleur naming global :+1: